src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::parseInt)
```

Comment:
Try to use a minimum amount of stream operations
```suggestion
        return numbers.stream()
                .flatMap(...)
                .mapToInt(Integer::parseInt)
```